doc/how-to/add_service.rst

```diff
@@ -0,0 +1,41 @@
+.. _howto-add-service:
+
+How to add a new service
```

Comment:
```suggestion
How to add a service
```
Sorry, pet peeve. ;)
How would you add an old service? :stuck_out_tongue_winking_eye: